### ISSUE TITLE
allow passing in client connection options (ie. timeout)

### ISSUE THREAD

**Issue by hoboman313** [tonybadguy/yelp-fusion#9]
^ would be nice to have. Currently there's no easy way to let the request timeout if yelp is taking too long to respond.

**Comment by tonybadguy** [comment]
ah good idea I'll look into it

**Comment by tonybadguy** [comment]
published to latest release (v2.2.1). Let me know if that works for you:

```
const client = yelp.client(apiKey, {
  socketTimeout: 5000
});
```
